angularSTEP/src/app/map/map.component.html

```diff
@@ -1,5 +1,5 @@
 
-<mat-card class="map-card">
+<!--<mat-card class="map-card">
   <form [formGroup]="mapsForm" class="location-form">
     <mat-form-field appearance="outline" class="full-width">
     <mat-label>Location</mat-label>
```

Comment:
This did not have to get commented out, I don't think. As long as the place that calls this component is commented out, it should be fine. I would undo this comment